content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,146 @@
+- Make DVC more accessible by hiding the complexity of the CLI
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+# Why Visual Studio Code
+
+One of [our core beliefs at Iterative](https://iterative.ai/why-iterative/) is
+that our products should couple tightly with existing tools and processes for
+data science projects. [VS Code is the IDE of choice for
+many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including many of us at Iterative. Here are a few reasons why we really like it:
+
+VS Code...
+
+- is open source and lightweight
+- has excellent support for Git workflows
+- supports Jupyter Notebooks
+- comes with remote development and execution built-in
+- has a rich ecosystem of extensions
+
+# Features
+
+Our first course of action was to include all DVC features related to data
+management and experiments in the extension. Here are the main features that are
+available when you install the extension right now:
+
+## DVC View: experiments, metrics, and plots
+
+An entirely new view to manage your experiments. Run new experiments, analyze
+their outcomes through plots and metrics, and compare them to find the best
+model.
+
+![Experiment tables to compare different
+experiments](/uploads/images/2022-06-14/dvc-view-experiments-table.png)
+
+![Plots to visualize your
+experiments](/uploads/images/2022-06-14/dvc-view-plots.png)
+
+## Command palette
+
+The DVC extension makes extensive use of the Command Palette, reducing the need
+to learn commands by heart:
+
+![Use the command palette to quickly find DVC
+features](/uploads/images/2022-06-14/command-palette.gif)
+
+## Source control management
+
+Manage changes to your datasets and models right from the source control panel,
+just like you would with changes to your code. Use `checkout`, `commit`, `add`,
+`push`, and `pull` straight from the interface:
+
+![Get a quick overview of changes to your data in the source control
+panel](/uploads/images/2022-06-14/source-control.png)
+
+<!--
+TODO: seems to overlap with Source control management; can we safely remove this?
+
+## Tracked resources
+
+Resources tracked by DVC can be viewed and managed through the tracked explorer
+view. See at a glance which datasets and models have been changed, and use
+`push` and `pull` to synchronize with your remote:
+
+![Tracked resources in the explorer
+view](/uploads/images/2022-06-14/tracked-resources.png) -->
+
+# What's next?
+
+We're excited that version 1.0 is live and available to the public! From here on
+out, we plan on bringing even more features to the extension, including:
+
+- Pipeline (DAG) support
+- [TPI](https://github.com/iterative/terraform-provider-iterative) integration
+  for remote execution of experiments
+- `dvc.yaml` syntax features, including autocomplete and snippets
+- Parallel coordinate plots
+
+Give us feedback on what you'd like to see next!
+
```

Comment:
Visually the post dwindles here because it all turns to bullets.  Maybe extend paragraph or add gif. (https://media.giphy.com/media/Tt9jctxaVjRny/giphy.gif or https://media.giphy.com/media/WhTC5v5qQP4yAUvGKz/giphy.gif)

Reply:
Added a little bit more filler sentences for visual padding and rewrote the _What's next_ list into a paragraph. Can tweak this further once we're sure the acknowledgements and resources are complete!